File: src/main/java/com/ptmanager/backend/labor/LaborCostReport.java
package com.ptmanager.backend.labor;

import java.time.LocalDate;
import java.util.List;

public record LaborCostReport(
    long workplaceId,
    LocalDate from,
    LocalDate to,
    long totalCost,
    List<EmployeeCost> employees
) {

    public record EmployeeCost(
        long employeeId,
        String name,
        long totalMinutes,
        int hourlyWage,
        long cost
    ) {
    }
}

File: src/main/kotlin/com/ptmanager/backend/payroll/dto/LaborCostReport.kt
package com.ptmanager.backend.payroll.dto

import java.time.LocalDate

data class LaborCostReport(
    val workplaceId: Long,
    val from: LocalDate,
    val to: LocalDate,
    val totalCost: Long,
    val employees: List<EmployeeCost>,
) {

    data class EmployeeCost(
        val employeeId: Long,
        val name: String,
        val totalMinutes: Long,
        val hourlyWage: Int,
        val cost: Long,
    )
}
